srml/session/src/lib.rs

```diff
@@ -236,10 +236,18 @@ decl_storage! {
 		Changed: bool;
 
 		/// The next key for a given validator.
-		NextKeyFor build(|config: &GenesisConfig<T>| {
+		pub NextKeyFor build(|config: &GenesisConfig<T>| {
```

Comment:
It would be better to keep this private and introduce a `get(next_key_for)` clause.